bundles/org.openhab.binding.tesla/src/main/resources/ESH-INF/thing/models.xml

```diff
@@ -14,7 +14,6 @@
 		<description>A Tesla Model S Vehicle</description>
 
 		<channels>
-			<channel id="allowwakeup" typeId="allowwakeup" />
```

Comment:
Was it unused, not linked? Does the channelType still exist?

Reply:
The channel type does no longer exist, resulting in a warning being logged when adding my Model S